requirements.txt

```diff
@@ -6,3 +6,5 @@
 #
 attrs==19.1.0
 python-dotenv==0.10.3
+six==1.12.0               # via tenacity
```

Comment:
`tenacity` should be only for `konfetti[vault]` and `konfetti[async-vault]` installations, pls, update `extras_requires` in `setup.py`

Reply:
And it should be removed from this place